atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -16,11 +16,13 @@
 from atomic_reactor.constants import (PLUGIN_BUMP_RELEASE_KEY, PROG, KOJI_RESERVE_MAX_RETRIES,
                                       KOJI_RESERVE_RETRY_DELAY)
 from atomic_reactor.config import get_koji_session
+from atomic_reactor.metadata import label
 from atomic_reactor.util import is_scratch_build
 from koji import GenericError
 import koji
 
 
+@label('koji-build-id')
```

Comment:
We probably don't need this anymore, @tkdchen's #1824 is close to done so we will have the label in the postbuild task

Reply:
ok well removed